ci/github-script/bot.js

```diff
@@ -576,7 +576,10 @@ module.exports = async ({ github, context, core, dry }) => {
 
   // Controls level of parallelism. Applies to both the number of concurrent requests
   // as well as the number of concurrent workers going through the list of PRs.
-  const maxConcurrent = 20
+  // We'll only boost concurrency when we're running many PRs in parallel on a schedule,
+  // but not for single PRs. This avoids things going wild, when we accidentally make
+  // too many API requests on treewides.
+  const maxConcurrent = context.eventName === 'pull_request' ? 1 : 20
```

Comment:
Don't we also need to account for `pull_request_target`? Or is this only handling the `test.yml` workflow?

Reply:
Entirely right, this is the wrong check to begin with. I'll want to do the same check that is done inside the rate limiter with `context.payload.pull_request`.